definition/IVisitor.ts

```diff
@@ -0,0 +1,13 @@
+
+export interface IVisitor {
```

Comment:
After deciding that we will use existing visitor framework instead of new visitors, I remove some code but some code missed was missed out. This is one such file. I will remove this.